src/components/credit.tsx

```diff
@@ -8,8 +8,8 @@ import { Design } from '@guardian/types/Format';
 import type { Option } from '@guardian/types/option';
 import { map, withDefault } from '@guardian/types/option';
 import { pipe2 } from 'lib';
-import React from 'react';
 import type { FC, ReactElement } from 'react';
+import React from 'react';
```

Comment:
I've just realised it might be possible to make this an explicit import of `Fragment` and then change the shirt hand `<>` to `<Fragment>`. This would meet the original goal of removing the full `import React from 'react';` statement. 

Something like 
```typescript
import { Fragment } from 'react';
```
might do it, but you'd have to test after

Reply:
Are you sure this is needed? I'm not seeing any issues running without this import 🤔

Reply:
Comes up with an error on mine when not imported

Reply:
What kind of errors were you seeing? Lint, compile or runtime?